packages/react/src/components/SparklineChartCard/SparklineChartCard.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { useMemo, useRef, useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { isEmpty, isEqual, defaultsDeep } from 'lodash-es';
+
+import Card from '../Card/Card';
+import { AreaChart } from '../AreaChart';
+import { CARD_SIZES } from '../../constants/LayoutConstants';
+import { CardPropTypes, SparklineChartPropTypes } from '../../constants/CardPropTypes';
+import { settings } from '../../constants/Settings';
+import { usePrevious } from '../../hooks/usePrevious';
+import { getUpdatedCardSize } from '../../utils/cardUtilityFunctions';
+
+const { iotPrefix } = settings;
+
+const propTypes = {
+  ...CardPropTypes,
+  ...SparklineChartPropTypes,
+};
+
+const defaultProps = {
+  size: CARD_SIZES.MEDIUMWIDE,
+  title: null,
+  content: {
+    xLabel: 'x label',
+    yLabel: 'y label',
+  },
+  footerContent: null,
+  locale: 'en',
+  values: [],
+  i18n: { noDataLabel: 'No data available' },
+};
+
+const SparklineChartCard = ({
+  title,
+  values,
+  content,
+  size,
+  breakpoint,
+  locale,
+  isLoading,
+  testId,
+  footerContent,
+  isExpanded,
+  i18n,
+  style,
+  ...others
+}) => {
+  const listRef = useRef(null);
+  const chartRef = useRef(null);
+
+  const mergedI18n = useMemo(() => ({ ...defaultProps.i18n, ...i18n }), [i18n]);
+
+  const newSize = useMemo(() => getUpdatedCardSize(size), [size]);
+  const contentWithDefaults = useMemo(() => defaultsDeep({}, content, defaultProps.content), [
+    content,
+  ]);
+
+  const isChartDataEmpty = isEmpty(values);
+
+  const options = useMemo(
+    () => ({
+      grid: {
+        x: {
+          enabled: false,
+        },
+        y: {
+          enabled: false,
+        },
+      },
+      axes: {
+        bottom: {
+          visible: false,
+          title: contentWithDefaults.xLabel,
+          mapsTo: contentWithDefaults.xProperty,
+          thresholds: contentWithDefaults.xThresholds,
+          scaleType: 'time',
+        },
+        left: {
+          visible: false,
+          title: contentWithDefaults.yLabel,
+          mapsTo: contentWithDefaults.yProperty,
+          thresholds: contentWithDefaults.yThresholds,
+          scaleType: 'linear',
+        },
+      },
+      color: contentWithDefaults.color,
+      points: {
+        enabled: false,
+      },
+      legend: {
+        enabled: false,
+      },
+      toolbar: {
+        enabled: false,
+      },
+    }),
+    [contentWithDefaults]
+  );
+
+  const [listHeight, setListHeight] = useState();
+  useEffect(() => {
+    setListHeight(listRef?.current?.clientHeight);
```

Comment:
Why is this needed?

Reply:
The list under the chart doesnt have a default height set, so I thought about having this to know the height the chart can take, I would expect a list with max 3/4 items
![image](https://user-images.githubusercontent.com/22034430/160410213-ee97c5e2-7e5a-4b9a-9849-93e8487ce9ac.png)
![image](https://user-images.githubusercontent.com/22034430/160410368-997938c8-4f45-4925-9336-b7c4ac8153a9.png)